packages/chakra-ui-docs/docs/accordion.mdx

```diff
@@ -1 +1,219 @@
-# Accordion
\ No newline at end of file
+import Vue from 'vue'
+import LiveEditor from '../components/LiveEditor'
+import { Accordion, AccordionItem, AccordionHeader, AccordionPanel, AccordionIcon, Box  } from '@chakra-ui/vue'
+import Code from '../components/CodeBlock'
+Vue.component('Accordion', Accordion)
+Vue.component('AccordionItem', AccordionItem)
+Vue.component('AccordionHeader', AccordionHeader)
+Vue.component('AccordionPanel', AccordionPanel)
+Vue.component('AccordionIcon', AccordionIcon)
+Vue.component('Box', Box)
+
+
+# Accordion
+
+Accordions display a list of high-level options that can expand/collapse to reveal more information.
+
+### import
+
+Chakra UI Vue exports 5 accordion related components:
+
+- `Accordion`: The wrapper that uses cloneElement to pass it's prop to the `AccordionItem`.
```

Comment:
I think we should modify this. You can say:
----
- `Accordion`: The wrapper that passes it's prop to all `AccordionItem` children